### DISCUSSION TITLE
Artist Artwork

### DISCUSSION THREAD

**Discussion by floydp386** [MoleMan1024/audiowagon#104] [General]
![20230123_162428_resized](https://user-images.githubusercontent.com/116339566/214104012-a9e82ade-a260-46e4-b82f-79643e4d13f2.jpg)
![20230123_165745_resized](https://user-images.githubusercontent.com/116339566/214104036-f6c14075-df55-4830-9625-70a2a789bf2a.jpg)

Is it normal that there is no artwork for Artists (see picture)?
Album art is working perfectly.
I'm using version 2.3.1

**Comment by floydp386** [comment]
Also I have already - ejected USB drive, deleted the corresponding database, then re-indexed metadata.

**Comment by MoleMan1024** [comment]
Yes, this is normal.
